src/Microsoft.DotNet.Compiler.Common/Executable.cs

```diff
@@ -88,7 +90,18 @@ private void MakeCompilationOutputRunnableForCoreCLR()
         private void CopyContentFiles()
         {
             var contentFiles = new ContentFiles(_context);
-            contentFiles.StructuredCopyTo(_runtimeOutputPath);
+
+            if (_compilerOptions.CopyToOutputInclude != null)
+            {
+                var resolver = new IncludeFilesResolver(_compilerOptions.CopyToOutputInclude);
+                var includeEntries = resolver.GetIncludeFiles(PathUtility.EnsureTrailingSlash(_runtimeOutputPath));
```

Comment:
i did a quick search on `IncludeFilesResolver`'s usage. all of them are use the `GetIncludeFiles` method only and then the resolver is released. can this be done in a static method?